sylvia-derive/src/multitest.rs

```diff
@@ -696,32 +702,64 @@ impl<'a> MultitestHelpers<'a> {
 
     fn generate_impl_contract(&self) -> TokenStream {
         let Self {
-            contract, custom, ..
+            contract,
+            custom,
+            override_entry_points,
+            ..
         } = self;
         let sylvia = crate_module();
 
-        #[cfg(not(tarpaulin_include))]
-        let migrate_body = if self.is_migrate {
-            quote! {
-                #sylvia ::cw_std::from_slice::<MigrateMsg>(&msg)?
-                    .dispatch(self, (deps, env).into())
-                    .map_err(Into::into)
+        let instantiate_body = override_entry_points
+            .get_entry_point(MsgType::Instantiate)
+            .map(OverrideEntryPoint::emit_multitest_dispatch)
+            .unwrap_or_else(|| {
+                OverrideEntryPoint::emit_multitest_default_dispatch(MsgType::Instantiate)
+            });
+
+        let exec_body = override_entry_points
+            .get_entry_point(MsgType::Exec)
+            .map(OverrideEntryPoint::emit_multitest_dispatch)
+            .unwrap_or_else(|| OverrideEntryPoint::emit_multitest_default_dispatch(MsgType::Exec));
+
+        let query_body = override_entry_points
+            .get_entry_point(MsgType::Query)
+            .map(OverrideEntryPoint::emit_multitest_dispatch)
+            .unwrap_or_else(|| OverrideEntryPoint::emit_multitest_default_dispatch(MsgType::Query));
+
+        let sudo_body = override_entry_points
+            .get_entry_point(MsgType::Sudo)
+            .map(OverrideEntryPoint::emit_multitest_dispatch)
+            .unwrap_or_else(|| {
+                quote! {
+                    #sylvia ::anyhow::bail!("sudo not implemented for contract")
+                }
+            });
+
+        let migrate_body = match override_entry_points.get_entry_point(MsgType::Migrate) {
+            Some(entry_point) => entry_point.emit_multitest_dispatch(),
+            None if self.is_migrate => {
+                OverrideEntryPoint::emit_multitest_default_dispatch(MsgType::Migrate)
             }
-        } else {
-            quote! {
+            None => quote! {
                 #sylvia ::anyhow::bail!("migrate not implemented for contract")
-            }
+            },
         };
 
-        #[cfg(not(tarpaulin_include))]
-        let reply = if let Some(reply) = self.reply.as_ref() {
-            quote! {
-                self. #reply((deps, env).into(), msg).map_err(Into::into)
-            }
-        } else {
-            quote! {
-                #sylvia ::anyhow::bail!("reply not implemented for contract")
-            }
+        let reply_body = match override_entry_points.get_entry_point(MsgType::Reply) {
+            Some(entry_point) => entry_point.emit_multitest_dispatch(),
+            None => self
+                .reply
+                .as_ref()
+                .map(|reply| {
```

Comment:
I don't think that you need those external curly braces, so if it is not becasue of some formatting issue I would get rid of them.

Reply:
It is because of the formatting unfortunately